base/applications/rapps/include/appinfo.h

```diff
@@ -63,7 +63,31 @@ IsInstalledEnum(INT x)
     return (x >= ENUM_INSTALLED_MIN && x <= ENUM_INSTALLED_MAX);
 }
 
+enum UninstallCommandFlags
+{
+    UCF_NONE =   0x00,
```

Comment:
```suggestion
    UCF_NONE   = 0x00,
```
(let's align the equal sign since we align the values too.)